sklearn/metrics/pairwise.py

```diff
@@ -799,9 +799,11 @@ def manhattan_distances(X, Y=None, *, sum_over_features=True):
     Parameters
     ----------
     X : array-like of shape (n_samples_X, n_features)
+        An array where each row is a sample and each column is a feature.
 
-    Y : array-like of shape (n_samples_Y, n_features), default=None
-        If `None`, uses `Y=X`.
+    Y : array-like of shape=(n_samples_Y, n_features), default=None
```

Comment:
By convention based on [Guidelines for writing documentation](https://scikit-learn.org/stable/developers/contributing.html#guidelines-for-writing-documentation), we do not use `=`.

```suggestion
    Y : array-like of shape (n_samples_Y, n_features), default=None
```

Reply:
Updated across my PRs. Will read through the guidelines as well. Thanks for bringing this to my attention!